web3/_utils/threads.py

```diff
@@ -79,10 +95,10 @@ def __init__(self, target=None, args=None, kwargs=None):
         self.args = args
         self.kwargs = kwargs
 
-    def run(self):
+    def run(self) -> None:
         self._return = self.target(*self.args, **self.kwargs)
 
-    def get(self, timeout=None):
+    def get(self, timeout: int=None) -> Any:
```

Comment:
`timeout` should probably be a `float`, same with all of the other timeouts and things that represent time units.

Reply:
What about timestamps? Shouldn't those be `int`'s?

Reply:
Yeah, block timestamps are ints.